.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - run:
           name: Get test corpus etag
           command: |
-            curl https://meta-scrubber-test-corpus.s3.us-west-1.amazonaws.com/exif-image-corpus.tar.gz -I | grep -Fi ETag | sed 's/ETag: //g' > /tmp/exif-image-corpus-etag
+            curl https://meta-scrubber-test-corpus.s3.us-west-1.amazonaws.com/exif-image-corpus.tar.gz -I | grep -Fi ETag | sed 's/ETag: //g' | sed 's/"//g' > /tmp/exif-image-corpus-etag
```

Comment:
Oh the test code actually expects quotes.  We can change that if we need to

Reply:
ooooh yeah I just realized that too - the thing that was actually causing the problems were the newlines at the end of that etag file. I believe I just finally got it working :grin:

Reply:
https://app.circleci.com/pipelines/github/getlantern/meta-scrubber/48/workflows/a391024d-3ab7-4824-b5a0-21377b2b585c/jobs/57/parallel-runs/0/steps/0-104

Reply:
Hey nice!  Looks good